src/blueprints/Do.Blueprints.Service.Application/CodingStyle/SingleByUnique/MarkActionAsSingleByUniqueConvention.cs

```diff
@@ -17,7 +17,12 @@ public void Apply(ActionModelContext context)
         if (context.Action.MappedMethod.TryGetSingle<SingleByUniqueAttribute>(out var unique))
         {
             context.Action.AdditionalAttributes
-                .Add($"{typeof(SingleByUniqueAttribute).FullName}(\"{unique.PropertyName}\", typeof({unique.PropertyType.FullName}))");
+                .Add($"{nameof(SingleByUniqueAttribute)}(\"{unique.PropertyName}\", typeof({unique.PropertyType.FullName}))");
         }
     }
+
+    public void Apply(ApiModelContext context)
+    {
+        context.Api.Usings.Add("Do.CodingStyle.SingleByUnique");
```

Comment:
move all usings to their feature under `ConfigureApiModel` instead of `ConfigureApiModelConventions`